GitFyle.Core.Api/Brokers/Storages/StorageBroker.ContributionTypes.References.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using GitFyle.Core.Api.Models.Foundations.ContributionTypes;
+using Microsoft.EntityFrameworkCore;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial class StorageBroker
+    {
+        void SetContributionTypeReferences(ModelBuilder modelBuilder)
+        {
+            int maxStringLength = 255;
+
+            modelBuilder.Entity<ContributionType>()
+                .Property(contributionType => contributionType.Name)
+                .HasMaxLength(maxStringLength)
+                .IsRequired();
```

Comment:
Do not make use of a variable to set the max length.  If you use the variable more than once it will lead to entanglement and you can unintendedly change other properties if you alter the value.  Rather do `.HasMaxLength(255)`

```suggestion
            modelBuilder.Entity<ContributionType>()
                .Property(contributionType => contributionType.Name)
                .HasMaxLength(255)
                .IsRequired();
```